lnrpc/rpc.proto

```diff
@@ -1367,12 +1369,17 @@ message Route {
     */
     uint32 total_time_lock = 1 [json_name = "total_time_lock"];
 
+    /**
+    delta calculated by subtracting current block height from the time_lock
```

Comment:
PR title says "paytoroute should use delta expiry", but no code changes for paytoroute (sendtoroute). It ignores delta time locks if specified?

Reply:
Agree with Joost, and you may also need to make changes to queryroutes to keep it compatible with sendtoroute.